server/messages.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package server;
+
+import "google/api/annotations.proto";
+
+message apiVersion{
```

Comment:
Please can we capitalise the first letter of message names as per the [protobuf style guide](https://developers.google.com/protocol-buffers/docs/style)